itests/src/test/java/org/apache/unomi/itests/migration/Migrate16xTo220IT.java

```diff
@@ -84,8 +84,16 @@ public void checkMigratedData() throws Exception {
         checkViewEventRestructured();
         checkEventTypesNotPersistedAnymore();
         checkForMappingUpdates();
+        checkNewIndexesExists();
```

Comment:
would be greate to have more info in the method name: checkEventSessionRollover2_2_0 something like this